tools/monorepo-utils/src/code-freeze/commands/branch/index.ts

```diff
@@ -20,10 +20,12 @@ import {
 } from '../../../core/github/repo';
 import { WPIncrement } from '../../../core/version';
 import { Logger } from '../../../core/logger';
-import { Options } from './types';
 import { getEnvVar } from '../../../core/environment';
 
-const getNextReleaseBranch = async ( options: Options ) => {
+const getNextReleaseBranch = async ( options: {
```

Comment:
I'd like to change these types later, owner and repo should be mandatory, but in a refactor I tried to do it ended down a rabbit hole. I've just inlined this one use of `Options` for now, but the plan would be to move to something like `const getNextReleaseBranch = async (owner: string, repo: string)`